atlasdb-jepsen-tests/build.gradle

```diff
@@ -32,14 +32,14 @@ dependencies {
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
     compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jdk8'
     compile group: 'com.google.guava', name: 'guava'
-    compile group: 'com.jayway.awaitility', name: 'awaitility'
     compile group: 'one.util', name: 'streamex'
     compile group: 'org.clojure', name: 'clojure'
     compile group: 'org.slf4j', name: 'slf4j-api'
 
     processor group: 'org.immutables', name: 'value'
 
     testCompile group: 'org.assertj', name: 'assertj-core'
+    testCompile group: 'org.hamcrest', name: 'hamcrest-library'
```

Comment:
Huh. This implies Awaitility included Hamcrest and we were including the matchers on our main classpath! I guess that kind of makes sense though, since they probably reuse the matchers for awaiting for various conditions.